Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -1330,3 +1330,81 @@ QWidget* qMRMLLayoutManager::viewWidget(vtkMRMLNode* viewNode) const
 
   return d->viewWidget(viewNode);
 }
+
+//------------------------------------------------------------------------------
+void qMRMLLayoutManager::pauseRender(bool pause)
```

Comment:
Since we have two methods (pause and resume render), I don't think we need the bool input argument.